common/lib/xmodule/xmodule/js/src/sequence/display.coffee

```diff
@@ -161,10 +162,15 @@ class @Sequence
 
       # update the data-attributes with latest contents only for updated problems.
       if @anyUpdatedProblems new_position
-        $.each @updatedProblems[new_position], (problem_id, latest_content) =>
+        $.each @updatedProblems[new_position], (problem_id, latest_data) =>
+          latest_content = latest_data[0]
```

Comment:
It's mostly for code readability.  My thinking was that defining a temporary variable in this case would make the semantics of it clearer.